packages/create-amplify/src/initial_project_file_generator.ts

```diff
@@ -80,11 +81,7 @@ export class InitialProjectFileGenerator {
 
     await this.executeWithDebugLogger(
       targetDir,
-      this.executableName === 'npm'
-        ? 'npx'
-        : this.executableName.startsWith('yarn')
-        ? 'yarn'
-        : this.executableName,
+      this.packageManagerProps.binaryRunner,
```

Comment:
the PackageManagerController should not expose it's binary name. Perhaps the PackageManagerController needs another method like `await packageManagerController.run(command, args)` where it runs the provided command in the idiomatic way for that package manager

Reply:
`run` might not be necessary now after the refactor. What do you think?